session9/src/com/itscool/homework/CpmparingEx4.java

```diff
@@ -0,0 +1,22 @@
+package com.itscool.homework;
+
+import java.util.Scanner;
+
+public class CpmparingEx4 {
```

Comment:
Typo in class name: I think it should have been `ComparingEx4`. You can rename it as follows: 
in the Project pane (left side) right click on the class name -> Refactor -> Rename -> <enter new correct name> -> Refactor (blue button). This will rename both java class and file name.